app/commands/unlock_all_command.rb

```diff
@@ -2,9 +2,14 @@
 class UnlockAllCommand < BaseCommand
   def run
     transaction do
-      active_locks = user.locks.active
+      # Gather an Array of Environment objects from each of the user's active locks.
+      environments = []
+      user.locks.active.each do |lock|
+        environments.push(lock.environment)
+      end
```

Comment:
More ruby'ish version of this is

``` ruby
environments = user.locks.active.map { |lock| lock.environment }
```

Reply:
Thanks, I'll sneak that into my next PR. I was actually trying to figure out how to do that!